.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+# match black formatter's behavior
+ignore = E203, W503
```

Comment:
It would be nice to include the names of the ignored rules. It'll be helpful to know what checks we're skipping.